Displays/PIXIE.txt

```diff
@@ -0,0 +1,117 @@
+          1: Graphics mode, 2 colors
+          2: Graphics mode, 4 colors
+          3: Graphics mode, 8 colors
+          4: Graphics mode, 16 colors
+
+       Graphics mode will use the same video_ram starting address as text mode,
+       but video_ram size and layout are different. Graphics mode will also use
+       the same palette as text mode, but only the colors 0-n (depending on the
+       color depth) are used. Fontram is not relevant in graphics mode.
+
+
+Graphics mode video ram:
+    a 128 by 96 screen is organized in 1-4 bitplanes of 768 words each.
+    Each bitplane consists of 96 rows of 8 words. Number of bitplanes
+    depends on the number of colors. 2 color mode has only one bitplane,
+    while 16 color mode has four bitplanes.
+
+              number  memory  total
+    bitplane  colors  offset  memory
+    --------  ------  ------  ------
+       0         2        0     768   2, 4, 8 and 16 color modes
+       1         4      768    1536   4, 8 and 16 color modes
+       2         8     1536    2304   8 and 16 color modes
+       3        16     2304    3072   16 color mode
+
+    Bitplane number is used as a bit position to refer the color palette.
+    The bitplane 0 bit is the least significant bit in LEM palette. 
+        e.g. if bitplane 0 is clear, but bitplanes 1-3 are set,
+             the palette color would be 0b1110 -> 0xE -> 14
+
+    Gaphics are stored to bitplane in a linear fashion; each word of data
+    on a line is located at an address that is one greater than the word on
+    its left. i.e.  Each line is a "plus one" continuation of the previous
+    line.
+                   _______________________________________
+                  |    |    |    |    |    |    |    |    |
+                  | 20 | 21 | 22 | 23 | 24 | 25 | 26 | 27 |
+                  |____|____|____|____|____|____|____|____|
+                  |    |    |    |    |    |    |    |    |
+                  | 28 | 29 | 30 | 31 | 32 | 33 | 34 | 35 |
+                  |____|____|____|____|____|____|____|____|
+                  |    |    |    |    |    |    |    |    |
+                  | 36 | 37 | 38 | 39 | 40 | 41 | 42 | 43 |
+                  |____|____|____|____|____|____|____|____|
+                   92 more rows like these
+                  |    |    |    |    |    |    |    |    |
+                  | 780| 781| 782| 783| 784| 785| 786| 787|
+                  |____|____|____|____|____|____|____|____|
+
+    The map represents a single bitplane (one bit of color) image at
+    word addresses 20 through 787. Each of these addresses accesses
+    one word (16 pixels) of a single bitplane. Image with 16 colors
+    has four bitplanes like this.
+
+Hardware ID
+    PIXIE hardware id is 0x774df615, while LEM1802 hardware id is
+    0x734df615. To guarantee full compatibility with all LEM1802
+    software, PIXIE has a dip switch located in the back side.
+
+    Dip swicth has two positions that can be used to change the
```

Comment:
Also, is the "graphics mode" locked out in position 0 (LEM1802), or is it still there?